src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let resultedString = '';
+
+    for (let i = 0; i < this.length; i += 1) {
+      const character = this[i];
+
+      if (character !== null && character !== undefined) {
+        resultedString += character;
+      }
```

Comment:
A good place to use [the nullish coalescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing).